### PR TITLE
Hampus/extern role on verify

### PR DESCRIPTION
**Pull request by Hampfh**

### CODE REVIEW

File: src/shared/utils/alias_to_year_mapper.ts [hunk 1]
```diff
@@ -3,16 +3,10 @@ import { AliasName } from "../alias-mappings";
...
-		case 1:
-			return AliasName.YEAR2;
-		case 2:
-			return AliasName.YEAR3;
-	}
-	return AliasName.CS_MASTER;
+	return (
+		[AliasName.YEAR1, AliasName.YEAR2, AliasName.YEAR3][currentYear] ??
+		AliasName.CS_MASTER
+	);
```
Comment (Herkarl) [inline src/shared/utils/alias_to_year_mapper.ts:11]: Unsure how idiomatic this is, but it should work

**Review by Herkarl — APPROVED** [review]
I don't know too much about the code base, so take my review with a grain of salt.
But it should work from what I can see.

File: src/commands/verify/subcommands/submit/verify-submit.handler.ts [hunk 2]
```diff
@@ -47,7 +48,6 @@ export const handleVerifySubmit = async (
 	}
 
 	verifiedUsers.set(discordId, emailAddress);
-	console.log("hello");
```
Comment (niklasvatn) [inline src/commands/verify/subcommands/submit/verify-submit.handler.ts:50]: :)

File: src/commands/verify/subcommands/submit/verify-submit.handler.ts [hunk 3]
```diff
@@ -58,7 +58,7 @@ export const handleVerifySubmit = async (
 		if (yearRole && year) {
 			await setYearRoles(user, yearRole, interaction.guild);
 			await handleChannelAlias(mapYearToAlias(year), interaction, joinChannel);
-		}
+		} else setExternRole(user, interaction.guild);
```
Comment (niklasvatn) [inline src/commands/verify/subcommands/submit/verify-submit.handler.ts:61]: I can't see why we would want to make this assumption really.
Is it easier to find all of this roles instead of looking for someone who is verified but does not have a year?

Anyway. It might make sense since we have years in hodis for everyone (or almost everyone I would assume) from 2017.

**Review by niklasvatn — APPROVED** [review]

File: src/shared/utils/alias_to_year_mapper.ts [hunk 4]
```diff
@@ -3,10 +3,16 @@ import { AliasName } from "../alias-mappings";
...
+	switch (current) {
+		case 0:
+			return AliasName.YEAR1;
+		case 1:
+			return AliasName.YEAR2;
+		case 2:
+			return AliasName.YEAR3;
+	}
+	return AliasName.CS_MASTER;
 }
```
Comment (niklasvatn) [inline src/shared/utils/alias_to_year_mapper.ts:18]: Just interested, why was this changed and reverted?

Comment (Hampfh) [reply src/shared/utils/alias_to_year_mapper.ts:18]: I'm not really a fan of switch cases and switched to an array-indexed approach but I got some feedback from Herman that it might have made the code more difficult to read so I reverted it back
